documentation/tutorial.md

```diff
@@ -563,33 +557,30 @@ export URI=mongodb://$DB_USERNAME:$DB_PASSWORD@$HOST_IP/$DB_NAME?replicaSet=$REP
 Relations, or what Juju documentation [describes as Integration](https://juju.is/docs/sdk/integration), are the easiest way to create a user for MongoDB in Charmed MongoDB. Relations automatically create a username, password, and database for the desired user/application. As mentioned earlier in the [Access MongoDB section](#access-mongodb) it is a better practice to connect to MongoDB via a specific user rather than the admin user.
 
 ### Data Integrator Charm
-Before relating to a charmed application, we must first deploy our charmed application. In this tutorial we will relate to the [Data Integrator Charm](https://github.com/canonical/data-integrator). This is a bare-bones charm that allows for central management of database users, providing support for different kinds of data platforms (e.g. MongoDB, MySQL, PostgreSQL, Kafka, etc) with a consistent, opinionated and robust user experience. In order to deploy the Data Integrator Charm we must clone its source code from GitHub and create the charm executable ourselves: 
+Before relating to a charmed application, we must first deploy our charmed application. In this tutorial we will relate to the [Data Integrator Charm](https://charmhub.io/data-integrator). This is a bare-bones charm that allows for central management of database users, providing support for different kinds of data platforms (e.g. MongoDB, MySQL, PostgreSQL, Kafka, etc) with a consistent, opinionated and robust user experience. In order to deploy the Data Integrator Charm we can use the command `juju deploy` we have learned above:
 
-<!--- note in the future replace this with juju deploy instead of from git --->
 ```shell
-git clone https://github.com/canonical/data-integrator.git
-cd data-integrator/
-sudo snap install charmcraft --classic
-charmcraft pack
+juju deploy data-integrator --channel edge --config database-name=test-database
 ```
-After packing the charm, you can see that a charm executable named `data-integrator_ubuntu-22.04-amd64.charm` has been created in the `data-integrator` directory. When we deploy the charm we can also specify the name of the database that we want created in MongoDB with the `database` config option. To deploy this charm with Juju and create a database in MongoDB named `test-database` enter:
-```shell
-juju deploy ./data-integrator_ubuntu-22.04-amd64.charm --config database=test-database
+The expected output:
+```
+Located charm "data-integrator" in charm-hub, revision 3
+Deploying "data-integrator" from charm-hub charm "data-integrator", revision 3 in channel edge on jammy
```

Comment:
We can remove ", revision 3" completely and put there "..." or alike. Up 2 you.
I would keep it AS IS, at least to show the reader which revision has been tested during tutorial preparation.

Reply:
Yes, I understand. I can see your point and that there is value on having the number in. However, we can probably also point this out in the text, saying 

"Note that, at the time of writing the revision published for the data-integrator charm was 3. This may change in the future, as newer revisions of the data-integrator charm are released".

Reply:
As you wish. Replaced with "..." to avoid loosing context while reading the tutorial.
We anyway have a revisions in the table "juju status" below. Resolved?